app/views/static/cloud_network/cloud-network-form.html.haml

```diff
@@ -0,0 +1,133 @@
+%form#form_div{'name'          => "angularForm",
+               'form-changed'  => true,
+               'miq-form'      => true,
+               'model'         => "vm.cloudNetworkModel",
+               'model-copy'    => "vm.modelCopy",
+               'ng-cloak'      => "",
+               'ng-show'       => "vm.afterGet"}
+
+  %h3
+    = _('Network Provider')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Network Manager')
+      .col-md-8{'ng-class' => "{'has-error': angularForm.ems_id.$invalid}", 'ng-if' => "vm.newRecord"}
+        %select{'name'                        => "ems_id",
+                'ng-model'                    => "vm.cloudNetworkModel.ems_id",
+                'ng-change'                   => "vm.filterNetworkManagerChanged(vm.cloudNetworkModel.ems_id)",
+                'ng-options'                  => 'ems.id as ems.name for ems in vm.available_ems',
+                'pf-select'                   => true,
+                'required'                    => "",
+                'selectpicker-for-select-tag' => ""}
+          %option{'value' => ""}
+            = "<#{_('Choose')}>"
+        %span.help-block{"ng-show" => "angularForm.ems_id.$error.required"}
+          = _("Required")
+      .col-md-8{'ng-if' => "!vm.newRecord"}
+        %input.form-control{'type'         => "text",
+                            'name'         => "ems_name",
+                            'ng-maxlength' => 128,
+                            'ng-model'     => "vm.cloudNetworkModel.ext_management_system.name",
+                            'ng-disabled'  => true}
+    %h3
+  .form-horizontal{'ng-if' => "vm.cloudNetworkModel.ems_id"}
+    %h3
+      = _('Placement')
+    .form-group
+      %label.col-md-2.control-label
+        = _('Cloud Tenant')
+      .col-md-8{'ng-class' => "{'has-error': angularForm.cloud_tenant_id.$invalid}", 'ng-if' => "vm.newRecord"}
+        %select{'name'                        => "cloud_tenant_id",
+                'ng-model'                    => "vm.cloudNetworkModel.cloud_tenant.id",
```

Comment:
You randomly renamed `cloud_tenant_id` to `cloud_tenant.id` here.

Ruby side is still doing

```
    options[:cloud_tenant] = find_record_with_rbac(CloudTenant, params[:cloud_tenant_id]) if params[:cloud_tenant_id]
```

So I assume that's a bug?